### ISSUE TITLE
Tutorials for BYOD

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/training-material#5295]
It would be nice to have a tutorial to show how people can integrate their own data pool (BYOD):

* S3
* Wedav (NextCloud, Owncloud)
* Dropbox
* maybe Onedata

**Comment by abretaud** [comment]
I guess we can always run an annotation workflow, like the [new helixer one](https://training.galaxyproject.org/training-material/topics/genome-annotation/tutorials/helixer/tutorial.html)

I think there are some genome to annotate from the Genome Ark remote source, which is S3 right? And it's always possible to save the result on some other S3 :shrug:

**Comment by bgruening** [comment]
> I think there are some genome to annotate from the Genome Ark remote source, which is S3 right? And it's always possible to save the result on some other S3 🤷

Yes that sounds great! We could also use Nextcloud from the EU - B2Drop to be available to most researchers?

**Comment by abretaud** [comment]
> We could also use Nextcloud from the EU - B2Drop to be available to most researchers?

Yes, why not!
Not a lot of bandwidth to work on that, but happy to discuss/help

**Comment by hexylena** [comment]
Would be *fantastic* to have some mini tutorials on this!! :) (Also for the new export process to various locations)
